packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/create/iti38-envelope.ts

```diff
@@ -28,14 +28,18 @@ function createSoapBody(bodyData: OutboundDocumentQueryReq): object {
   const classCode = bodyData.classCode;
   const practiceSettingCode = bodyData.practiceSettingCode;
   const facilityTypeCode = bodyData.facilityTypeCode;
-  const serviceDateFrom = dayjs(bodyData.serviceDate?.dateFrom).format("YYYYMMDDHHmmss");
-  const serviceDateTo = dayjs(bodyData.serviceDate?.dateTo).format("YYYYMMDDHHmmss");
-  const documentCreationDateFrom = dayjs(bodyData.documentCreationDate?.dateFrom).format(
-    "YYYYMMDDHHmmss"
-  );
-  const documentCreationDateTo = dayjs(bodyData.documentCreationDate?.dateTo).format(
-    "YYYYMMDDHHmmss"
-  );
+  const serviceDateFrom = bodyData.serviceDate?.dateFrom
```

Comment:
only setting these dates via dayjs if they are defined. Otherwise dayjs sets them to the current date

Reply:
this is resolving a bug in the DQ code where it would only query for documents for the current date. [Context](https://metriport.slack.com/archives/C0616FCPAKZ/p1715110549778259?thread_ts=1715044583.086939&cid=C0616FCPAKZ)